bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerOpenOp.java

```diff
@@ -226,8 +225,12 @@ public String toString() {
                 public void readLastConfirmedComplete(int rc,
                         long lastConfirmed, Object ctx) {
                     if (rc != BKException.Code.OK) {
-                        closeLedgerHandle();
-                        openComplete(bk.getReturnRc(BKException.Code.ReadException), null);
+                        closeLedgerHandleAsync().whenComplete((r, ex) -> {
+                            if (ex != null) {
+                                log.error("Ledger {} close failed", ledgerId, ex);
```

Comment:
change replace log to LOG ,like this:
`LOG.error("Ledger {} close failed", ledgerId, ex);`